### ISSUE TITLE
Accessibility: Illustrated Message Component 

### ISSUE THREAD

**Issue by Sanjay-1312** [SAP/fundamental-ngx#4950]
Accessibility: Illustrated Message Component (There are couple of accessibility issues found while testing, so i have attached error document for you reference.)
Screenshot:
[Illustrated Message_Core_ARIBA Controls.docx](https://github.com/SAP/fundamental-ngx/files/6161326/Illustrated.Message_Core_ARIBA.Controls.docx)

**Comment by olshansky** [comment]
Changed estimate to 2 since it has 7 issues in one card and needs additional investigation about native behavior of email/phone links.

**Comment by olshansky** [comment]
- [ ] **Issue 1:** within ‘structure’ header, press tab to navigate to the buttons

**Observed Behavior:**
- There is no text alternative or tooltip provided for the buttons available.
The speech output is:

    > Button Button- 
    > To activate press spacebar.
    > Button Button- 
    > To activate press spacebar.

**Expected Behavior:**
- Ideally, all the non textual elements like icons, symbols etc. should have a text alternative or tooltip provided and the same information should be read by the screen reader on navigation.

____

- [ ] **Issue 2:** within ‘No new mail’ header, press tab to navigate to the Compose button.

**Observed Behavior:**
- Two additional tab stops are provided for ‘compose’ button on navigating using tab key. i.e. on first tab focus navigates to ‘inbox’ and on the second tab focus navigates to ‘compose button’. Therefore, additional tab stops are provided.
 The speech output is:

  > Inbox 
  > Compose Button- 
  > To activate press spacebar.

**Expected Behavior:**
- Ideally, no additional tab stops should be available on navigation and the information read should be clear and precise. Therefore, the name ‘inbox’ should be associated to the button and should not have a separate tab stop.

____

- [ ] **Issue 3:** Within ‘dialog’ header, navigate to ‘open dialog illustrated message’ button and activate it

**Observed Behavior:**
- The text, ‘Unable to load data Check your internet connection. If that’s not it, try refreshing the page.’ Cannot be accessed using arrow keys when the dialog appears on the screen. i.e. the user cannot navigate within the text using keyboard.

**Expected Behavior:**
- Ideally, all the non interactive elements or static elements should be accessible using arrow keys. Therefore, the user should be able to navigate within the text in the dialog using arrow keys.

____

- [ ] **Issue 4:** Activate the ‘Open Dialog illustrated Message’ and hover the mouse to ‘Close’ button.

**Observed Behavior:**
- There is no text alternative or tooltip provided for the ‘Close’ button.

**Expected Behavior:**
- Ideally, all the non textual elements like icons, symbols etc. should have a text alternative or tooltip provided.

____

- [ ] **Issue 5:** 

**Observed Behavior:**
- On accessing the inbox, the tab stop is provided for the inbox header region.

**Expected Behavior:**
- Ideally, there shouldn’t be additional tab stop provided for the header region.

**Comment by olshansky** [comment]
**Issue 1:** buttons have a textual label, so that's not "non-textual elements". Changed text to more meaningful.

**Issue 2:** removed interaction from heading. 

**Issue 3:** 
> all the non interactive elements or static elements should be accessible using arrow keys. 

I don't think so, it doesn't work like that even on https://www.w3.org/WAI/. Point me at the specification if I'm wrong.
But I've added a few a11y enhancements for dialog.

**Issue 4:** looks like the button has a text alternative `aria-label="close"` ("close" is exactly text alternative for icon), but added additional description with tooltip (`title="Close dialog"`).

**Issue 5:** the same as **Issue 2**

**Comment by Sanjay-1312** [comment]
Issue 1: Fixed.

Issue 2: 
![image](https://user-images.githubusercontent.com/78714518/121864015-1039cf80-cd1a-11eb-964e-2c3c5ce04208.png)
OB: The tab chain is removed and the tab is given to button but dialog details is not read.
EB: On navigating to the 'Compose' button, the  dialog header 'Inbox' followed by the text element such as 'No new Email check again later and followed by the button name 'Compose' button details should be read at one go.

Issue 3:
![image](https://user-images.githubusercontent.com/78714518/121864990-18463f00-cd1b-11eb-9e6a-353187bbc918.png)
OB: using JAWS in VPC mode the text should be reachable using the arrow keys.
EB: The text within the dialog need to reachable and read by using arrow keys.
To toogle between the PC and VPC mode in jAWS use insert+z .

Issue 4
OB: The Tool tip is provided as 'Close Dialog' and the tool to information should be read.
![image](https://user-images.githubusercontent.com/78714518/121865916-16c94680-cd1c-11eb-8c9f-415145cabe0c.png)
EB: The tool tip information shouldn't b read only only 'Close' button should be read.

Issue 5: fixed

**Comment by Sanjay-1312** [comment]
Issue 2
Fixed

Issue 3:
Fixed



Issue 4:
![image](https://user-images.githubusercontent.com/78714518/123457016-1fb1f600-d601-11eb-9c3d-0cd480497d89.png)
OB: On activating the button the focus is available on 'Close' button
EB: on activating the button the dialog and dialog header name header should be read and the first UI element 'Reload' the focus should be available, 

JAWS 2021
URL: https://deploy-preview-5642--fundamental-ngx.netlify.app/#/core/illustrated-message

**Comment by olshansky** [comment]
@Sanjay-1312 but the first focusable item is "X" close button, so which button should be initially focused?

**Comment by Sanjay-1312** [comment]
@olshansky The focus should be available on the 'Reload' button. Using Esc if the dialog closes the focus is not required for 'Close' button.

**Comment by Sanjay-1312** [comment]
Issue 4: Fixed

URL:  https://deploy-preview-5642--fundamental-ngx.netlify.app/#/core/illustrated-message
JAWS 2021
